modules/theme/src/themes/default/modules/modal.variables

```diff
@@ -70,7 +70,7 @@
 --------------------------------*/
 @attributeModalContentHeight: 500px;
 @transferComponentSegmentHeight: 450px;
-@transferComponentListSegmentHeight: 300px;
+@transferComponentListSegmentHeight: 350px;
```

Comment:
This change might affect other places where we use the transfer component. Just FYI @inthirakumaaran

Reply:
Only in two instances we have used this variable, both are related to attributes